SwiftWisdom/Core/StandardLibrary/Array/Array+Utilities.swift

```diff
@@ -54,3 +54,43 @@ extension Array {
         }
     }
 }
+
+extension CollectionType {
+    public var ip_middleValues: [Generator.Element] {
```

Comment:
I use it in finding the median (after sorting an array), but I made it public because I imagined it might be nice to have an easy way to find the element(s) at the middle of an array.